en/docs/develop-components/develop-proxy/develop-a-git-api-proxy.md

```diff
@@ -0,0 +1,166 @@
+2. If you already have one or more components in your project, click **+ Create**. Otherwise, proceed to the next step.
+3. Click the **API Proxy** card. This opens the **Create an API Proxy** pane.
+4. You will see the three options to create a proxy component and here we select github option (selected by default).
+5. Specify the following values as component details:
+
+    !!! info
+        The **Component Name** field must be unique and cannot be changed after creation.System will generate a unique name for you. But you are free to change the generated name.
+        **Component Display Name** is a required field.
+
+    | **Field**       | **Value**                                  |
+    |-----------------|--------------------------------------------|
+    | **Component Display Name**| `Department Service`                    |
+    | **Component Name**        | `departmentService`                    |
+    | **Description**           | `This is a sample pet store`     |
+   
+6. Go to the **GitHub** tab.
+    - Click **Authorize with GitHub** to connect your GitHub account. If you haven’t connected your GitHub repository to Choreo, enter your GitHub credentials and select the repository you forked in the prerequisites section to install the [Choreo GitHub App](https://github.com/marketplace/choreo-apps).
+
+    !!! note
+        The **Choreo GitHub App** requires the following permissions:
+         - Read and write access to code and pull requests.
+         - Read access to issues and metadata.
+        
+        You can [revoke access](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-authorized-integrations#reviewing-your-authorized-github-apps) if needed. Write access is only used for sending pull requests; Choreo will not push changes directly to your repository.    
+
+6. Enter the following repository details:
+
+    | **Field**              | **Value**          |
+    |------------------------|--------------------|
+    | **Organization**       | Your GitHub account|
+    | **Repository**         | choreo-samples     |
+    | **Branch**             | **`main`**         |
+    | **API Directory**      | /choreo-samples/proxy-from-github/department-service |
+
+7. Specify the following values as API proxy details:
+
+    !!! info
+        The **Context** field must be unique and cannot be changed after creation.**Version** and **Target** are mandatory fields. **Target** can be changed at any time after the creation.
+
+    | **Field**       | **Value**                                  |
+    |-----------------|--------------------------------------------|
+    | **Context**     | `department-service/v1`                                   |
+    | **Version**     | `V1.0`                                      |
+    | **Target**      | `https://samples.choreoapps.dev/company/hr/department`|
+
+6. Click **Create**. This creates the API proxy component and takes you to the **Overview** page.
```

Comment:
This took me to the **Build** page.

Is it supposed to take me to the **Overview** page?

Reply:
This should be corrected to `Build`